package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -0,0 +1,474 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
rst stub for this?

Reply:
I've got one [here](https://github.com/MDAnalysis/mdanalysis/pull/2622/files#diff-0f3a2ec710c389155f8e7a2261d553b4); is it not showing up?